cmd/inferconfig/testdata/expected/circleci-demo-python-django.yml

```diff
@@ -10,12 +10,10 @@ jobs:
       - image: cimg/python:3.8-node
     steps:
       - checkout
-      - python/install-packages:
-          args: --dev
-          pkg-manager: pipenv
+      - python/install-packages
       - run:
           name: Run tests
-          command: pipenv run python manage.py test
+          command: python manage.py test
```

Comment:
Have we confirmed this passes with https://github.com/CircleCI-Public/circleci-demo-python-django?

Reply:
you're right I need to add that back in, I got too trigger happy with the deletions here